ground/gcs/src/plugins/pathplanner/flightdatamodel.cpp

```diff
@@ -517,6 +601,33 @@ bool FlightDataModel::getHomeLocation(double *homeLLA) const
 }
 
 /**
+ * @brief ModelUavoProxy::setHomeLocation Take care of scaling the home location UAVO to
+ * degrees (lat lon) and meters altitude
+ * @param [out] home A 3 element double array containing the desired home location
+ * @return True if successful, false otherwise
+ */
+bool FlightDataModel::setHomeLocation(double *homeLLA)
+{
+    ExtensionSystem::PluginManager *pm = ExtensionSystem::PluginManager::instance();
+    UAVObjectManager * objMngr = pm->getObject<UAVObjectManager>();
```

Comment:
Correct. Checks are missing in a lot of places for this type of issues, look at the getHomeLocation as well, old code that's just as bad - I'd like to push this PR through, then open another issue to fix all those bad practices afterwards, to clearly separate bug fixes/features from code health ? What do you think ?